core/src/events.rs

```diff
@@ -336,11 +336,13 @@ pub enum KeyCode {
     Backspace = 8,
     Tab = 9,
     Return = 13,
+    Command = 15,
     Shift = 16,
     Control = 17,
     Alt = 18,
     Pause = 19,
     CapsLock = 20,
+    Numpad = 21,
```

Comment:
Did you mean `NumLock` here?

Reply:
Oh no, it's a modifier, or something like that, sorry: https://help.adobe.com/en_US/air/reference/html/flash/ui/Keyboard.html#NUMPAD
It was just weird at a first glance.

Reply:
I did the same double take. It's possible we could give it a different name.

Reply:
Or perhaps a comment like `// A meta-key used as modifier, not NumLock` would clear it up.